src/http/api/notification.ts

```diff
@@ -49,12 +42,12 @@ export function createGetNotificationsHandler(
             });
         }
 
-        const account = await accountService.getDefaultAccountForSite(
+        const account = await this.accountService.getDefaultAccountForSite(
```

Comment:
If you're changing this line we might as well use `ctx.get('account')` here too - nabd tho